src/types/eatery_type.py

```diff
@@ -16,6 +16,7 @@ class EateryType(ObjectType):
   about_short = String(required=True)
   campus_area = Field(CampusAreaType, required=True)
   coordinates = Field(CoordinatesType, required=True)
+  collegetown = Boolean(required=True)
```

Comment:
think this is a reasonable use case for a non required variable, confirm with ios maybe